pkgs/servers/http/apache-httpd/2.4.nix

```diff
@@ -2,6 +2,7 @@
 , nixosTests
 , proxySupport ? true
 , sslSupport ? true, openssl
+, tlsSupport ? false, rustls-ffi, Foundation
```

Comment:
I am not super happy with those two option names since ssl and tls are often used interchangeably and openssl so far also had tls support.

Reply:
Well it's the name of Apache module: https://httpd.apache.org/docs/2.4/mod/mod_tls.html

I have used this name to be consistent with what we currently have. Maybe something like `modTlsSupport` would be clearer?

Reply:
Hmmm, I see. Renaming to modTlsSupport seems like a good option.

Reply:
Done, thanks for the feedback.